cli/src/cli-info/CliInfo.ts

```diff
@@ -3,5 +3,6 @@
  */
 export interface CliInfo {
 	appName: string;
+	commandName?: string;
```

Comment:
I think `commandName` is basically the same as `appName`, thus I suggest to use `appName` instead of adding `commandName`. What do you think?

Reply:
Creating another key was a sort of defensive measure against breaking other possible uses of `appName` but I understand that it might be unnecessary 👍